tests/NovelRT.Tests/CMakeLists.txt

```diff
@@ -51,24 +36,54 @@ set(SOURCES
   Utilities/EventTest.cpp
 
   main.cpp
- )
+)
+
+if(NOVELRT_BUILD_INTEROP)
+    set(INTEROPBUILD 1)
+    set(SOURCES ${SOURCES}
+      Interop/NovelRTInteropUtilsTest.cpp
+      Interop/Ecs/NrtCatalogueTest.cpp
+      Interop/Ecs/NrtComponentBufferMemoryContainerTest.cpp
+      Interop/Ecs/NrtComponentCacheTest.cpp
+      Interop/Ecs/NrtEntityCacheTest.cpp
+      Interop/Ecs/NrtSparseSetMemoryContainerTest.cpp
+      Interop/Ecs/NrtSystemSchedulerTest.cpp
+      Interop/Maths/GeoBoundsTest.cpp
+      Interop/Maths/GeoMatrix4x4Test.cpp
+      Interop/Maths/GeoVector2FTest.cpp
+      Interop/Maths/GeoVector3FTest.cpp
+      Interop/Maths/GeoVector4FTest.cpp
+      Interop/Maths/QuadTreePointTest.cpp
+      Interop/Maths/QuadTreeTest.cpp
+      Interop/SceneGraph/NovelRTSceneNodeTest.cpp
+      Interop/Timing/NovelRTTimestampTest.cpp
+    )
+endif()
 
 add_executable(Engine_Tests ${SOURCES})
+
+if(NOVELRT_BUILD_INTEROP)
+    copy_build_products(Engine_Tests
+      DEPENDENCY Engine
+      TARGET_LOCATION $<TARGET_FILE_DIR:Engine_Tests>
+
+      DEPENDENCY Interop
```

Comment:
This should be a condition (`CONDITION TARGET Interop`) but right now it will fail. Remind me to fix it later.

Reply:
Created issue as a reminder